lib/preprocessors/glimmer.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+const { getTemplateLocals } = require('@glimmer/syntax');
+const {
+  preprocessEmbeddedTemplates,
+} = require('ember-template-imports/lib/preprocess-embedded-templates');
+const util = require('ember-template-imports/src/util');
+
+const TRANSFORM_CACHE = new Map();
+const TEXT_CACHE = new Map();
+
+function gjs(text, filename) {
+  const isGlimmerFile = filename.endsWith('.gjs') || filename.endsWith('.gts');
+
+  if (!isGlimmerFile) {
+    return [
+      {
+        filename,
+        text,
+      },
+    ];
+  }
+
+  if (text.includes(util.TEMPLATE_TAG_PLACEHOLDER)) {
```

Comment:
Can you add a comment explaining why we bail out here?

Reply:
done